File: java-hangman-app/src/main/java/ru/polina_project/hangman/util/HangmanStage.java
package ru.polina_project.hangman.util;

public class HangmanStage {
    public static final String[] STAGES = {
            """
                        +-----+
                        |     |
                        O     |
                       /|\\    |
                       / \\    |
                              |
                        =========
            """,
            """
                        +-----+
                        |     |
                        0     |
                       /|\\    |
                         \\    |
                              |
                        =========
            """,
            """
                        +-----+
                        |     |
                        0     |
                       /|\\    |
                              |
                              |
                        =========
            
            """,
            """
                        +-----+
                        |     |
                        O     |
                        |\\    |
                              |
                              |
                        =========
            """,
            """
                        +-----+
                        |     |
                        O     |
                        |     |
                              |
                              |
                        =========
            
            """,
            """
                        +-----+
                        |     |
                        O     |
                              |
                              |
                              |
                        =========
            """,
            """
                        +-----+
                        |     |
                              |
                              |
                              |
                              |
                        =========
            """,
            """
                              +
                              |
                              |
                              |
                              |
                              |
                        =========
            """
    };
}

File: kotlin-hangman-app/src/main/kotlin/ru/polina_project/hangman/util/HangmanStage.kt
package ru.polina_project.hangman.util

object HangmanStage {

    val STAGES = listOf(
        """
                        +-----+
                        |     |
                        O     |
                       /|\    |
                       / \    |
                              |
                        =========
            
            """
            .trimIndent(),
        """
                        +-----+
                        |     |
                        0     |
                       /|\    |
                         \    |
                              |
                        =========
            
            """
            .trimIndent(),
        """
                        +-----+
                        |     |
                        0     |
                       /|\    |
                              |
                              |
                        =========
            
            """
            .trimIndent(),
        """
                        +-----+
                        |     |
                        O     |
                        |\    |
                              |
                              |
                        =========
            
            """
            .trimIndent(),
        """
                        +-----+
                        |     |
                        O     |
                        |     |
                              |
                              |
                        =========
            
            """
            .trimIndent(),
        """
                        +-----+
                        |     |
                        O     |
                              |
                              |
                              |
                        =========
            
            """
            .trimIndent(),
        """
                        +-----+
                        |     |
                              |
                              |
                              |
                              |
                        =========
            
            """
            .trimIndent(),
        """
                              +
                              |
                              |
                              |
                              |
                              |
                        =========
            
            """
            .trimIndent()
    )
}


